lib/quby/questionnaires/dsl/questionnaire_builder.rb

```diff
@@ -54,6 +54,12 @@ def sbg_key(sbg_key)
           @questionnaire.sbg_key = sbg_key
         end
 
+        def sbg_domain(sbg_key, options = {})
+          domain = default_sbg_domain_options.merge(options.assert_valid_keys(:begin, :end, :outcome, :key, :primary))
```

Comment:
Omdat Ruby variabelen met de namen `begin` en `end` niet zo leuk vindt. We kunnen ze eventueel hernoemen naar `starts_at` en `ends_at` oid.